src/components/Checkbox.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <label :for="id" class="checkbox-label" :class="labelClasses">
+    <input
+      :id="id"
+      type="checkbox"
+      class="checkbox"
+      :checked="checked"
+      :name="inputName"
+      :value="inputValue"
+      :disabled="disabled"
+      @change="onChange"
+    /><span>
+      <slot>
+        {{ label }}
+      </slot>
+    </span>
```

Comment:
I get why prettier auto-formats things this way because of Vue's whitespace handling, but in this case the whitespace means nothing, so for readability's sake I think it would be better to put each tag on a separate line.

```suggestion
    />
    <span>
      <slot>
        {{ label }}
      </slot>
    </span>
```